node-js-basics-101/node_modules/ws/lib/Constants.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const safeBuffer = require('safe-buffer');
+
+const Buffer = safeBuffer.Buffer;
+
+exports.BINARY_TYPES = ['nodebuffer', 'arraybuffer', 'fragments'];
+exports.GUID = '258EAFA5-E914-47DA-95CA-C5AB0DC85B11';
+exports.EMPTY_BUFFER = Buffer.alloc(0);
```

Comment:
__[jshint]__
* 'exports' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->